pulser-simulation/pulser_simulation/simulation.py

```diff
@@ -402,7 +402,7 @@ def set_initial_state(
         if isinstance(state, str) and state == "all-ground":
             self._initial_state = qutip.tensor(
                 [
-                    self.basis["d" if self._interaction == "XY" else "g"]
+                    self.basis["u" if self._interaction == "XY" else "g"]
```

Comment:
Thinking about it, I find it weird to have |0> -> "u" (for up ?) and |1> -> "d" (for down I guess) in the code. Won't it make more sense to invert the order of the "u" and "d" when building self.basis in _build_basis_and_op_matrices ?